wdl-lint/tests/lints/command-shellcheck-ok/source.errors

```diff
@@ -0,0 +1,49 @@
+note[SectionOrdering]: sections are not in order for task `test1`
+   ┌─ tests/lints/command-shellcheck-ok/source.wdl:7:6
```

Comment:
You can except the rules that aren't relevant in these examples to make it just check the relevant ones.

Reply:
My expectation from the name is that actually no diagnostics would be in this file.

Reply:
Agreed, just some messiness I haven't cleaned up yet.